### DISCUSSION TITLE
How to make a specific view public while keeping others private?

### DISCUSSION THREAD

**Discussion by hhimanshu** [community/community#8962] [Projects and Issues] [answered]
Hello folks, 

First of all, THANK YOU for Issues Beta! You've done a fantastic job in simplifying a mess that many other products in similar domain tend to carry.
I have a situation where I want a single board that contains 2 types of repositories - public and private.
I want to work with my team on a single board where work is managed internally, however, I want to create a single view which I can make publicly accessible on the internet. This public view only contains public repository (using filters, group by milestone)

Currently, I am managing both type of repositories on a single board without any issues. However, I want to make a view publicly accessible (not the entire project).

Does anyone has any recommendation on how to achieve this?

**Comment by Katsute** [comment, marked as answer]
Any issues/pulls on private repositories are hidden from the public, see GitHub's project table as an example: https://github.com/orgs/github/projects/4826

As an alternative you could also just have two separate project tables.

**Reply by hhimanshu** [reply]
Wow! That did not occur to me, very thoughtful. Thank you @Katsute
